external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -229,118 +160,150 @@
           SourceFiles="@(LauncherFolderContents)"
           DestinationFolder="$(TestHostRootPath)\Launcher\%(RecursiveDir)"
           SkipUnchangedFiles="true"/>
+
   </Target>
 
   <Target Name="VerifyMatchingProjectNVersions"
           BeforeTargets="Build">
+
     <Error Condition="'$(ProjectNTfsExpectedPrerelease)' != '$(ProjectNTfsTestILCExpectedPrerelease)'" Text="The versions of ProjectN Targeting pack and TestILC have gone out of sync. Please make sure both of the prerelease versions on dependencies.props match." />
+
   </Target>
 
   <Target Name="CopyTestILCToolsToTestHost"
           AfterTargets="AfterResolveReferences"
-          Condition="'$(TargetsAOT)'=='true'">
+          Condition="'$(TargetsAOT)' == 'true'">
+
     <PropertyGroup>
       <TestILCToolsPackageName>TestILC</TestILCToolsPackageName>
       <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x64'">$(TestILCToolsPackageName).amd64ret</TestILCToolsPackageName>
       <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x86'">$(TestILCToolsPackageName).x86ret</TestILCToolsPackageName>
       <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'arm'">$(TestILCToolsPackageName).armret</TestILCToolsPackageName>
       <TestILCFolder Condition="'$(TestILCFolder)' == ''">$(PackagesDir)$(TestILCToolsPackageName)\$(ProjectNTfsTestILCPackageVersion)\TestILC</TestILCFolder>
-      <TestILCFolder>$(TestILCFolder.Replace('/', '\'))</TestILCFolder>
     </PropertyGroup>
 
     <Warning Text="Could not find the TestILC Tools at '$(TestILCFolder)'. Please run sync." Condition="('$(TestILCFolder)' == '' or !Exists('$(TestILCFolder)')) AND '$(BuildAllConfigurations)' != 'true'" />
 
-    <ItemGroup Condition="'$(TestILCFolder)'!=''">
-      <TestILCFolderContents Include="$(TestILCFolder)\**\*" />
-    </ItemGroup>
-
-    <Copy Condition="'$(TestILCFolder)'!=''"
-          SourceFiles="%(TestILCFolderContents.FullPath)"
-          DestinationFiles="$(TestHostRootPath)\TestILC\%(TestILCFolderContents.RecursiveDir)%(TestILCFolderContents.Filename)%(TestILCFolderContents.Extension)"
+    <Copy Condition="'$(TestILCFolder)' != ''"
+          SourceFiles="$(TestILCFolder)"
+          DestinationFolder="$(InternalTestILCFolder)"
           SkipUnchangedFiles="true" />
 
   </Target>
 
-  <Target Name="AddTestPlatformDependenciesNetCore" BeforeTargets="ResolveReferences" Condition="'$(TargetGroup)' == 'netcoreapp'">
-    <Error Condition="!Exists('$(PackagesDir)$(XUnitRunnerPackageId)\$(XUnitPackageVersion)\tools\netcoreapp2.0\$(XUnitRunner).dll')"
-            Text="Error: looks the package $(PackagesDir)$(XUnitRunnerPackageId)\$(XUnitPackageVersion) not restored or missing $(XUnitRunner).dll."
+  <Target Name="AddTestPlatformDependencies" BeforeTargets="ResolveReferences">
```

Comment:
I presume this is what you were talking about in standup: you could instead consider just copying the generated props/targets from this depproj to some place with a known name.

eg: copy artifacts\obj\XUnit.Runtime\netcoreapp\XUnit.Runtime.depproj.nuget.g.* to some known location (potentially with a better name).

If you do this you'd need to be more careful about which packages participate, perhaps adding ExcludeAssets="build" in cases where we don't want to use the targets that came with the package.